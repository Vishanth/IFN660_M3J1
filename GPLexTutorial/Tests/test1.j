int foo2 = 0;

if (foo2 < 31)
{
    foo2 = foo2 + 42 + 9002l;
	foo3 = 1_23L + 300 + 1_0_5L;
}