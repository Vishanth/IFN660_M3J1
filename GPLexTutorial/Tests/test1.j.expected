INT
IDENT (foo2)
'='
NUMBER (0)
';'
IF
'('
IDENT (foo2)
'<'
NUMBER (31)
')'
'{'
IDENT (foo2)
'='
IDENT (foo2)
'+'
NUMBER (42)
'+'
INTEGERLITERAL (9002l)
';'
IDENT (foo3)
'='
INTEGERLITERAL (1_23L)
'+'
NUMBER (300)
'+'
INTEGERLITERAL (1_0_5L)
';'
'}'
EOF
